### DISCUSSION TITLE
setTranslation(): Argument #2 ($value) must be of type string, null given

### DISCUSSION THREAD

**Discussion by Denysetya1** [tomatophp/filament-translations#35] [Q&A]
When I tried to translate into Indonesian using Google Translator, the queue job failed, when I checked for error, it said

TypeError: TomatoPHP\FilamentTranslations\Models\Translation::setTranslation(): Argument #2 ($value) must be of type string, null given, called in C:\laragon\www\happytopup\vendor\tomatophp\filament-translations-google\src\Jobs\ScanWithGoogleTranslate.php on line 38 and defined in C:\laragon\www\happytopup\vendor\tomatophp\filament-translations\src\Models\Translation.php:47
Stack trace:

**Comment by fadymondy** [comment]
hi @Denysetya1, please note that the Google Translation is a Crawling, not an official API, which means it may now work with you because of the reCap of Google or rate limited.
